### ISSUE TITLE
Accessibility: Missing heading tag

### ISSUE THREAD

**Issue by vikrampm1** [woothemes/woocommerce#51893]
**Prerequisites (mark completed items with an [x]):**
- [x] I have checked that my issue type is not listed here https://github.com/woocommerce/woocommerce/issues/new/choose
- [x] My issue is not a security issue, support request, bug report, enhancement or feature request (Please use the link above if it is).

### Issue Description:

This content is not tagged as an HTML heading, though it is one. If content is not appropriately tagged as headings, it will not show in the headings list for screen reader users. Additionally, it may make it more difficult for blind and visually impaired users to understand the structure of the page or to jump to this section.  

![Image](https://github.com/user-attachments/assets/8a98d1be-8eb9-49be-b583-f684f1c90345)

### Component or Page: 

Page: Cart (Modern)     

### Name affected element:

Cart Totals heading

### Relevant code snippets:

`<span class="wp-block-woocommerce-cart-order-summary-heading-block wc-block-cart__totals-title">Cart totals</span>`

### Recommended fix:

Wrap content that serves as a title or subtitle with a heading tag. The recommended heading level for this content (Cart Totals) is: H2.
